apps/student-portal/src/features/chats/components/ChatSidebar.tsx

```
@@ -0,0 +1,351 @@
+
+									return (
+										<div
+											key={conversation.id}
+											className={cn(
+												"w-full cursor-pointer overflow-hidden rounded-lg border p-2.5 transition-all hover:bg-muted/50",
+												selectedConversationId === conversation.id
+													? "border-primary bg-primary/5 shadow-sm"
+													: "border-transparent bg-card/50",
+											)}
+											onClick={() => onSelectConversation?.(conversation.id)}
+										>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)